jib-core/src/main/java/com/google/cloud/tools/jib/docker/DockerClient.java

```diff
@@ -158,6 +158,38 @@ public String load(ImageTarball imageTarball) throws InterruptedException, IOExc
     }
   }
 
+  /**
+   * Saves an image tarball from the Docker daemon.
+   *
+   * @see <a
+   *     href="https://docs.docker.com/engine/reference/commandline/save/">https://docs.docker.com/engine/reference/commandline/save</a>
+   * @param imageReference the image to save
+   * @param outputPath the destination path to save the output tarball
+   * @return stdout from {@code docker}
```

Comment:
Do we need this? For a successful save, `docker save` doesn't output anything.

Reply:
Yeah I guess not.